controllers/src/test/scala/cromwell/pipeline/controller/SecurityDirectiveTest.scala

```diff
@@ -1,18 +1,19 @@
-package cromwell.pipeline.datastorage.dao.repository.auth
+package cromwell.pipeline.controller
 
 import java.time.Instant
 
 import akka.http.scaladsl.model.StatusCodes
 import akka.http.scaladsl.model.headers.RawHeader
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.testkit.ScalatestRouteTest
-import cromwell.pipeline.datastorage.utils.auth.SecurityDirective._
-import cromwell.pipeline.datastorage.utils.auth.{ AccessTokenContent, AuthContent, SecurityDirective }
+import cromwell.pipeline.controller.utils.auth.SecurityDirective
+import cromwell.pipeline.utils.auth.{ AccessTokenContent, AuthContent }
 import cromwell.pipeline.utils.{ AuthConfig, ExpirationTimeInSeconds }
 import org.scalatest.{ Matchers, WordSpec }
 import pdi.jwt.algorithms.JwtHmacAlgorithm
 import pdi.jwt.{ Jwt, JwtAlgorithm, JwtClaim }
 import play.api.libs.json.Json
+import cromwell.pipeline.controller.utils.auth.SecurityDirective._
```

Comment:
Better to put this line after line #9

Reply:
oops.. .thats formatter's work

Reply:
have done this

Reply:
Nope, this is not yet done

Reply:
sorry, I can not realize how did I not mentioned that(!) Now it is alright!